File: Padaria-App/src/app/_models/Pedido.ts
import { Venda } from './Venda';
import { Produto } from './Produto';

export interface Pedido {

    id: string ;
    valorTotal: number ;
    dataInicio?: Date ;
    dataFinal?: Date ;
    qrcode: string ;
    clienteId: string ;
    vendas: Produto[];

}

File: Padaria.Domain/Entities/Pedido.cs
using Padaria.Domain.Entities.Base;
using System;
using System.Collections.Generic;

namespace Padaria.Domain.Entities
{
    public class Pedido : EntityBase
    {
        public double ValorTotal { get; set; }
        public DateTime? DataInicio { get; set; }
        public DateTime? DataFinal { get; set; }
        public string Qrcode { get; set; }
        public Cliente Cliente { get;}//deve readonly
        public List<Venda> Vendas { get; set; }

        
    }
}